packages/ffe-datepicker-react/src/calendar/ClickableDate.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import {
+    bool,
+    func,
+    number,
+    object,
+    oneOfType,
+    shape,
+    string,
+} from 'prop-types';
+import classNames from 'classnames';
+import moment from 'moment';
+
+export default class ClickableDate extends Component {
+    componentDidMount() {
+        this.focusIfNeeded();
+    }
+
+    componentDidUpdate() {
+        this.focusIfNeeded();
+    }
+
+    focusIfNeeded() {
+        const { date, forceFocus, activeRef } = this.props;
+        if (date.isFocus && forceFocus) {
+            activeRef.current.focus();
+        }
+    }
+
+    dateClassName() {
+        const { date, isBlured } = this.props;
+        const { isEnabled, isFocus, isSelected, isToday } = date;
+
+        return classNames({
+            'ffe-calendar__date': true,
+            'ffe-calendar__date--today': isToday,
+            'ffe-calendar__date--focus': isFocus && !isBlured,
+            'ffe-calendar__date--disabled': !isEnabled,
+            'ffe-calendar__date--selected': isSelected,
+            'ffe-calendar__date--disabled-focus': !isEnabled && isFocus,
+        });
+    }
+
+    tabIndex() {
+        return this.props.date.isFocus ? 0 : -1;
+    }
+
+    render() {
+        const {
+            date,
+            headers,
+            onClick,
+            language,
+            activeRef,
+            month,
+        } = this.props;
+        const year = moment(date.timestamp).format('yyyy');
+        const momentDate = moment(date);
+        const dayOfMonth = momentDate.format('D');
+        const monthName = language === 'en' ? month : month.toLowerCase();
+
+        return (
+            <td
+                aria-disabled={!date.isEnabled}
+                aria-selected={date.isSelected}
+                className="ffe-calendar__day"
+                headers={headers}
+                ref={activeRef}
```

Comment:
Skulle denne ref'en egentlig vært på `button`-elementet? Siden vi programmatisk setter fokus på den. Og sånn sett også fjernet `tabIndex={0}` fra `td`.

Reply:
Etter å ha sjekket mer her så tror jeg faktisk det er bedre å ha klikk-eventet på `td` og ikke en egen knapp inne i `td`. Dette siden det blir veldig mye nøsting inne tabellen som gjør at skjermlesere leser opp tabellen veldig rart. Har også sjekket andre som har valgt å droppe knapp inn i `td`. 
Alternativet er å lage custom "tabell" med div-er og knapper med tilhørende roller som da gjør det til en tabell

Reply:
Ja det blir kanskje vel mye å lese opp med mye forskjellig nøsting. Klarer vi å få fram for skjermlesere at td'en er klikkbar?

Reply:
Siden vi bruker `aria-selected=true/false` så tror jeg det skal være nok.

Reply:
Muligens. Litt usikker på om man egentlig må formidle det vha. en `role`. Men `td` har jo en innebygd `role` jeg ikke tror vi har lyst å overstyre. Men du får teste litt :)